lib/presentation/pages/home/partial/club_list_widget.dart

```diff
@@ -1,36 +1,89 @@
 import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+import 'package:liga_inggris_mobile/app/config/app_colors.dart';
+import 'package:liga_inggris_mobile/presentation/common/clickable_card_widget.dart';
 import 'package:liga_inggris_mobile/presentation/common/club_logo_widget.dart';
+import 'package:liga_inggris_mobile/presentation/controllers/home/home_controller.dart';
 import 'package:liga_inggris_mobile/services/club/model.dart';
 
-class ClubListWidget extends StatelessWidget {
+class ClubListWidget extends GetView<HomeController> {
   final List<ClubModel> clubs;
+  final VoidCallback onTap;
 
-  const ClubListWidget({super.key, required this.clubs});
+  const ClubListWidget({super.key, required this.clubs, required this.onTap});
+
+  void _handleTapCard(ClubModel club) {
+    controller.goToClubDetail(club.id ?? 0);
+  }
 
   @override
   Widget build(BuildContext context) {
-    return SingleChildScrollView(
-      scrollDirection: Axis.horizontal,
-      child: Row(
-        children: clubs.map((club) {
-          return Container(
-            margin: const EdgeInsets.all(8.0),
-            child: Card(
-              child: Column(
-                mainAxisSize: MainAxisSize.min,
-                children: [
-                  ClubLogoWidget(
-                    imageUrl: club.logo!,
-                    width: 100,
-                    height: 100,
+    return Column(
+      children: [
+        const Text(
+          "Club List",
+          style: TextStyle(fontSize: 18),
+        ),
+        SingleChildScrollView(
+          scrollDirection: Axis.horizontal,
+          child: Row(
+            children: [
+              ...clubs.map(
+                (club) {
+                  return ClickableCard(
+                    onTap: () => _handleTapCard(club),
+                    color: Colors.transparent,
+                    elevation: 0,
+                    child: DefaultTextStyle(
+                      style: TextStyle(color: AppColors.text),
+                      child: Container(
+                        margin: const EdgeInsets.symmetric(vertical: 4),
+                        child: SizedBox(
+                          height: 62,
+                          width: 62,
+                          child: Container(
+                            margin: const EdgeInsets.all(4),
+                            decoration: BoxDecoration(
+                                gradient: AppColors.homeLogoBackground,
+                                shape: BoxShape.circle),
```

Comment:
```suggestion
                                shape: BoxShape.circle,),
```
kasih coma, trus save auto format biar dak gantung